> ![image](https://private-user-images.githubusercontent.com/28448503/361037448-4724ddd8-f5b4-4b26-9886-79fd6c5a22d2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ0NjkzNjcsIm5iZiI6MTcyNDQ2OTA2NywicGF0aCI6Ii8yODQ0ODUwMy8zNjEwMzc0NDgtNDcyNGRkZDgtZjViNC00YjI2LTk4ODYtNzlmZDZjNWEyMmQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI0VDAzMTEwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY3ZTVjMGY1NDdhMzI1NWZlNjU0NjMyYWYzM2RmNGY5ODljYTNjZWQ4OWU1ZDdiNmFkYzA4NjJjYmY1ZGU3NGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Nrdc_YDAO1HmNspScqSckrhco2G4DxMo3P3RV5CtgNE)
> 
> That's fine—those are just normal warnings from the CFLAGS, and I can turn them off if you want. If there were any issues, an error would be displayed, and Conway wouldn't be built. However, that's not the case here, so if you check, you'll see that Conway was built successfully. The Makefile is meant to simplify the build process instead of doing it manually. Also, I noticed that you used 'chmod +x conway'; it's not a script, so there's no need to do that.
> 
> CFLAGS = -Wall -Wextra -pedantic -std=c99
> 
> -Wall: Enables most common warning messages during compilation. This helps catch potential issues in the code. -Wextra: Enables additional warning messages that are not included with -Wall. These warnings are more specific and can help identify even more potential issues. -pedantic: Enforces strict adherence to the C standard (in this case, C99). It generates warnings for any code that is not strictly compliant with the standard. -std=c99: Specifies that the code should be compiled according to the C99 standard, which is a version of the C programming language standard published in 1999.

Hmmmm, I get it but the output on your screen is different from mine, I get 

make: *** [Makefile:15: conway.o] Error 1

and the conway file is not generated.

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 CC = gcc
-CFLAGS = -Wall -Wextra -pedantic -std=c99
+CFLAGS = -Wall -Wextra -pedantic -std=gnu99
 LDFLAGS = -lncurses -lm
 
 SRC = conway.c

